scheduler/dev-config.edn

```diff
@@ -1,6 +1,17 @@
 {:port 12321
  ;; We'll set the user to vagrant, since that's the default for many Vagrant-based Mesos setups
  :authorization {:one-user "vagrant"}
+
+ ;; What function should be used to perform user authorization?
+ ;; See the docstring in cook.authorization for details.
+ :authorization-fn cook.authorization/configfile-admins-auth
```

Comment:
Consider having `:authorization-config` or some better named key which is a map. That map has a `:fn` key and we pass the map into our authorization-fn instead of passing the full config file.